advanced_html_css/accessibility/the_web_content_accessibility_guidelines_wcag.md

```diff
@@ -19,7 +19,7 @@ It's important to understand that while the WCAG can be incredibly helpful for b
 
 The WCAG are organized around four, core principles that should be kept in mind when implementing any sort of accessibility feature:
 
-1. **Perceivable:** Users must be able to perceive the information or user interfaces being presented. For example, Light text on a light background could be difficult for some users with a visual impairment to perceive.
+1. **Perceivable:** Users must be able to perceive the information or user interfaces being presented. For example, light text on a light background could be difficult for some users with a visual impairment to perceive.
```

Comment:
Nice catch on the capitalization here, thanks!

Reply:
i have updated this patch with your requested review changes, please let me know if there is anymore changes required, thanks :)

Reply:
Thanks ab!